timemory/roofline/__main__.py

```diff
@@ -112,7 +119,7 @@ def plot(args):
             _roofline.plot_roofline(ai_ranks[0], op_ranks[0], band_labels, args.display,
                                     fname, args.format, fdir, args.title,
                                     args.plot_dimensions[0], args.plot_dimensions[1],
-                                    args.plot_dimensions[2])
+                                    args.plot_dimensions[2], _inst_roofline)
```

Comment:
We should pass in the rtype and handle the logic there. I'd prefer to be able to set this boolean inside roofline.py, not in the main